Code/CSharpOOP3/Flowers.cs

```diff
@@ -0,0 +1,34 @@
+﻿namespace CSharpOOP3
+{
+    public class Flowers  //Task_01
+    {
+        public string Color //(setter-only property)
+        {
+            set { Color = value; }
+        }
+
+        public double Price { get { return Price; } }  //(read-only property)
+
+        public string Name//(Task_03)
```

Comment:
you will get infinitive recursion here :)
please add field _name to achieve the result

Reply:
done